addons/stock/tests/test_move.py

```diff
@@ -3564,7 +3564,10 @@ def test_scrap_2(self):
             'product_uom_id':self.product_consu.uom_id.id,
             'scrap_qty': 1,
         })
+        sequence = self.env['ir.sequence'].search([('code', '=', scrap._name), ('company_id', '=', scrap.company_id.id)])
+        number_next_actual = sequence.number_next_actual
         scrap.do_scrap()
+        self.assertEqual(number_next_actual, sequence.number_next_actual, 'sequence should not be changed after do_scrap')
```

Comment:
i don't understand, the idea is to get a sequence when calling do scrap, here you assert the opposite?